python/zed/zed.py

```diff
@@ -46,8 +46,8 @@ def create_pool(self, name, layout={'order': 'desc', 'keys': [['ts']]},
 
     def load(self, pool_name_or_id, data, branch_name='main',
              commit_author=getpass.getuser(), commit_body=''):
-        pool = urllib.parse.quote(pool_name_or_id)
-        branch = urllib.parse.quote(branch_name)
+        pool = urllib.parse.quote_plus(pool_name_or_id, safe='')
+        branch = urllib.parse.quote_plus(branch_name, safe='')
```

Comment:
My first comment about this was confusing. Apologies for that.

`quote_plus()` replaces a space with `+`. While that seems to work fine here, it isn't really what we want, which is to replace a space with `%20`. `quote(string, safe='')` will do that.

```suggestion
        pool = urllib.parse.quote(pool_name_or_id, safe='')
        branch = urllib.parse.quote(branch_name, safe='')
```